components/Navbar/Navbar.tsx

```diff
@@ -60,10 +123,34 @@ const Navbar = () => {
               </div>
             </div>
           </div>
+          <div className='relative'>
+            <button
+              className='relative text-white focus:outline-none rounded-full p-2'
+              onClick={handleToggleNotifications}
+            >
+              <Image
+                src='/images/bell.svg'
+                alt='notifications'
+                width={40}
+                height={40}
+              />
+              {hasNotifications && (
+                <span className='absolute top-0 right-0 bg-orange-500 text-white rounded-full w-4 h-4 flex items-center justify-center text-xs'></span>
+              )}
+            </button>
+            {hasNotifications && showNotifications && (
+              <div
+                ref={containerRef}
+                className='absolute right-0 mt-2 bg-white text-gray-800 rounded-lg shadow-lg p-4 max-h-60 overflow-y-auto '
+              >
+                {displayedNotifications?.map((notification) => (
+                  <Notification notification={notification} />
+                ))}
+              </div>
```

Comment:
instead of having this div, you should out this inside a `NotificationList` or `NotificationMenu` component. The reason for that is to separate concerns, the navbar doesn't need to know about the notification menu, it only toggles it. The CSS and any behavior inside the notification menu is a concern of the menu itself